app/src/main/java/com/projectit3048c/dao/iFoodDAO.kt

```diff
@@ -5,13 +5,13 @@ import retrofit2.Call
 import retrofit2.http.GET
 import retrofit2.http.Query
 
-interface iFoodDAO {
+interface IFoodDAO {
 
-    @GET("fdc/v1/foods/search?query=egg&dataType=Foundation&pageSize=3&pageNumber=1&sortBy=dataType.keyword&sortOrder=asc&api_key=trqHcOXD3WWuC2z5OhYoMiUpZaFp2GFtEcUSs8fA")
-    fun getAllFoods() : Call<Food>
+    @GET("/anastasiiaef/JSON/food")
+    fun getAllFoods() : Call<ArrayList<Food>>
```

Comment:
Is this from a different branch???

Reply:
Yes, we couldn't find an open JSON data and I created a JSON file on GitHub and read the file through the my-JSON-server.